pkg/controller/networkaddonsconfig/networkaddonsconfig_controller.go

```diff
@@ -498,6 +495,97 @@ func (r *ReconcileNetworkAddonsConfig) stopTrackingObjects() {
 	r.statusManager.SetFromPods()
 }
 
+func updateObjectsLabels(crLables map[string]string, objs []*unstructured.Unstructured) error {
+	var err error
+	for _, obj := range objs {
+		labels := obj.GetLabels()
+		if labels == nil {
+			labels = map[string]string{}
+		}
+		// Label objects with version of the operator they were created by
+		labels[cnaov1.SchemeGroupVersion.Group+"/version"] = operatorVersionLabel
+
+		labels, err = updateObjLabels(obj, labels, crLables)
```

Comment:
I don't think it's a good name.
the whole function is called `updateObjectsLabels`, so calling the sub-function `updateObjLabels` is not specific enough..
IIUC What this function specifically does is update the app-labels that are originated from the CNAO CR, if exists.. so the function name should be specific enough to say just that.